docs/starter-pages/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+	
+	<title>Techne Starter Pages</title>
+
+	<link rel="icon" type="image/png" href="images/favicon-96x96.png" sizes="96x96">
+
+  	<link rel="stylesheet" href="../public/dist/css/techne.min.css">
+  	<link rel="stylesheet" href="css/demo.css">
+
+</head>
+<body>
+
+	<!-- top nav -->
+	<div class="top-nav navbar navbar-fixed-top">
+		<h1 class="brand pull-left">Techne Starter Pages</h1>
+		<a href="http://www.hybris.com" target="_blank" class="brand pull-right">
+	    	<img class="brand-logo" src="images/hybris.svg" alt="hybris - an SAP Company"/>
+		</a>
+	</div>
+
+	<!-- sub nav  -->
+	<div class="top-nav navbar navbar-fixed-top sub-nav">
+		<a href="table.html" target="content-frame">Table</a>
+		<a href="cards.html" target="content-frame">Cards</a>
+		<a href="details-1.html" target="content-frame">Detail 1</a>
+		<a href="details-2.html" target="content-frame">Detail 2</a>
+	</div>
+
+	<div class="container-fluid offcanvas-row">
+		
+		<!-- left nav  -->
+		<div class="side-bar offcanvas-sidebar"></div>
+
+		<!-- content frame -->
+		<div class="content offcanvas-content">
+			
+			<iframe src="table.html" frameborder="0" width="100%" height="1000px" name='content-frame' class="content-frame"></iframe> 
```

Comment:
the height here doesn't seem quite right. Try using the same styles that Yaas uses